surveys/2024-annual-survey/translations/de.md

```diff
@@ -667,7 +667,7 @@ Type: select one
 - < 3 Jahre
 - < 5 Jahre
 - < 10 Jahre
-- > 10 Jahre
+- \> 10 Jahre
```

Comment:
This is currently rendered as Markdown block quote here on GitHub

(actually affects all translations?)

Reply:
Is this change needed? AFAICS the list of options seems to be correctly rendered (unless it was fixed on our platform when we inputted the data)

<details><summary>Screenshot from the survey</summary>
<p>

![grafik](https://github.com/user-attachments/assets/c3620dab-a3d2-486e-81f6-04747c361de7)


</p>
</details>

Reply:
Not strictly needed if you copy the text here from the code and not there rendered version.
The issue was more with Markdown rendering (e.g. here on GitHub) where `> ...` is shown as block quote:
![Rendered Markdown screenshot](https://github.com/user-attachments/assets/cc390467-c45a-4b01-9a7c-c680a6d3b604)

I will revert it again. If this change is desired (and does not complicate copying the text from the source), might be better to apply it to all translations then.